### PR TITLE
Completed Newsweek clone

### PR DESCRIPTION
**Pull request by DennisOsido**
Live Preview: https://rawcdn.githack.com/DennisOsido/Newsweek-Clone/9a80820c7def71b396d5ddb5876e4156a6302c53/index.html

## EDIT after the third review:
-  Removed Horizontal Scrollbar and Added Media Queries for Debate Section

## EDIT after the second review:
- Added Media Queries and Styles for Responsiveness with emphasis for the following screen sizes: 320px, 768px, and 1024px as recommended by TSE

## EDIT after the first review:
- Removed all inline styling
- "The Debate", "Newsletter" and "Subscribe" sections created piece by piece
- Fixed Navbar alignment and Footer Section
- Fixed "Editor's Pick" section

## In this solo project I have:
- Created a replica of Newsweek.com using Bootstrap classes wherever possible.

## Steps:
- Created a Header
- Created a Navbar
- Created 5 Sections
- Created a Footer section
- Added Bootstrap classes

## Requirements that were completed:
- Bootstrap classes were used wherever possible
- Made sure that page layout looks exactly like the source page
- Made sure that page layout does not break on smaller screens

### CODE REVIEW

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires changes.

File: index.html
Comment (pelzolga123) [inline index.html]: No inline styling, all styles should be inside styles.css file.

**Comment by pelzolga123** [comment]
## Hi Dennis!  
### Your project looks :+1: great!
Here are the things you need to change before this project can be approved:
* [ ] This elements should be created piece by piece not an image:
 image

image

image

* [ ] Try to find font, similar to original website, right now it looks very different. Also try to replicate similar font weight and size to make a better Newsweek clone. 

### Tablet version
* [ ] Navbar should look more like original, all elements should be aligned properly:
image
Original:
image
 * [ ] Editor's pick section should change like original:
image
 
Original:
image
 * [ ] Footer should look more like original:
image

Original:
image

### Mobile version
* [ ] It has white line on the right side, that should be removed:
image

Original:
image

* [ ] Sections should be displayed in next order:
* Top Story
* Featured Stories
* Opinion
* The Debate
* More Stories
* Culture & Travel
* Latest News


### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by DennisOsido** [comment]
## After the first review:
- Removed all inline styling
- "The Debate", "Newsletter" and "Subscribe" sections created piece by piece
- Fixed Navbar alignment and Footer Section
- Fixed "Editor's Pick" section

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Under Review By Tolulope Olusakin(TSE)
Please check on this in a few minutes for some feedback! :+1:

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Great work putting this together @DennisOsido :+1: 
Please consider my comment below towards tying this all together, make the necessary changes and request another review. I wish you the best!

#### Responsiveness
Your project could use a little more responsiveness, please look a little more closely at the sizes of elements, padding of sections, margins, and general element positionings. These are very key as they will affect the look, feel, and more importantly the usability of your sites. Please consider(at least) the following screen sizes: 320px, 768px, and 1024px, use your browser developer tools.

Try to make each section look as close to the original site as possiible

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Comment by DennisOsido** [comment]
## After the second review:

- Added Media Queries and Styles for Responsiveness with emphasis for the following screen sizes: 320px, 768px, and 1024px as recommended by TSE

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello Team!,
### You have done a good job so far and for also implementing changes from previous reviews :clap: However, you have a few changes to make before your project could be approved.
- Remove the horizontal scroll bar on mobile devices, the width needs to fit well
- Also, in the debate section, redesign to fit well on medium and small screens
medium screen
### Make these changes and resubmit for another code review. Good luck.
- Happy coding!

**Comment by DennisOsido** [comment]
## After the third review:
-  Removed Horizontal Scrollbar and Added Media Queries for Debate Section

**Comment by rOluochKe** [comment]
Hello Team!  
### Good job and for also making changes from previous reviews. You project looks good and I therefore, approve of it :boom: :100:
### you can merge and submit a project completion form from your dashboard. Good luck with the next project.
- Happy coding.

**Pull request merged**
